ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -374,7 +379,7 @@ export function PreviewConcierge(props: any) {
           if (type === GUEST_CHECK_IN) {
             getHostToGuestBus().next({
               type: HOST_CHECK_IN,
-              payload: { editMode: false, highlightMode }
+              payload: { editMode: false, highlightMode, rteConfig: rteConfig ?? {}, site: siteId }
```

Comment:
- We must resend rte config if it changes.
- Why are you sending the siteId now?